adapters/src/server/mod.rs

```diff
@@ -0,0 +1,263 @@
+use crate::{Catalog, Controller, ControllerConfig, ControllerError};
```

Comment:
This server should really be a separate crate from `adapters`

Reply:
That was my first instinct, but at the moment the server is a really thin layer on top of the controller API, really just an HTTP binding to the controller API.  I'll just feature gate it.